### DISCUSSION TITLE
Gift Card Email Notification

### DISCUSSION THREAD

**Discussion by coderreco** [swellstores/community#578] [Help]
In the [gift card model](https://developers.swell.is/backend-api/gift-cards/create-a-gift-card#:~:text=.-,send_email,-string) for non-physical gift cards, it shows the `send_email` and `send_note` fields for notifying. 

Using the frontend api, when I add the gift card product to my cart do I just pass those values in at the same time?

Thanks in advance.

**Comment by ericingram** [comment]
@coderreco Yes that's correct
